xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.cpp

```diff
@@ -27,7 +27,7 @@
   "" \
   "import " MODULE "\n" \
   "class xbmcout:\n" \
-  "  def __init__(self, loglevel=" MODULE ".LOGNOTICE):\n" \
+  "  def __init__(self, loglevel=" MODULE ".LOGINFO):\n" \
```

Comment:
Is this correct or should it be LOGWARNING?

Reply:
Looks ok to me.